src/System.Net.NameResolution/tests/PalTests/project.json

```diff
@@ -24,12 +24,16 @@
     "System.Text.RegularExpressions": "4.2.0-beta-24403-05",
     "System.Threading": "4.0.12-beta-24403-05",
     "System.Threading.Tasks": "4.0.12-beta-24403-05",
+    "System.Net.Sockets": {
```

Comment:
I'm not sure I understand the newly introduced issue: what was the change causing the issue?

To add some background: our PAL and UnitTests have the production code built-in by design. The design allows testing individual units instead of e2e API to avoid complicated multi-machine setup and ensure that hard to reach code is exercised.
System.Net.Internals contains types that expose more functionality compared to the ones with the same name exposed by the contracts. It is used to avoid leaking platform-specific implementation details into the public APIs and, in certain cases, to avoid cyclic dependencies between the contracts.